spec/yaml/swiftnav/sbp/gnss.yaml

```diff
@@ -17,6 +17,28 @@ include:
 
 definitions:
 
+ - MeGnssSignal:
```

Comment:
You don't need to introduce this new structure, since its identical to the `GnssSignal` structure. I think it's ok to reuse the `GnssSignal` structure with a slightly enhanced interpretation of the `sat` field in your new message structure.